Library/ViewModels/PledgeAmountViewModel.swift

```diff
@@ -6,12 +6,18 @@ import ReactiveSwift
 
 public typealias PledgeAmountData = (amount: Double, min: Double, max: Double, isValid: Bool)
 
+public typealias PledgeAmountViewConfigData = (
```

Comment:
Curious why not make this a struct instead of a tuple?

Reply:
We've just usually used tuples unless these get larger than 6 elements. If they're 7+ then one needs a struct for synthesized `Equatable` conformance. This seems somewhat supported by the [docs](https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html):

<img width="718" alt="image" src="https://user-images.githubusercontent.com/3735375/85428945-cb2df380-b532-11ea-808c-8667ad284879.png">